USERS.md

```diff
@@ -165,6 +165,11 @@ Users (Alphabetically)
       U: CiliumNetworkPolicy, Hubble, ClusterMesh
       L: https://clickhouse.com
 
+    * N: Cloutomate
+      D: Cloutomate uses Cilium as CNI for itself and customer installations
+      U: Networking Observability and Security, Service Mesh, Cluster Mesh
+      L: https://cloutomate.de
+
```

Comment:
can you split this change into a separate PR, so that it will be easier and faster to review and merge ? Thanks

Reply:
ddone